git-tag/README.md

```diff
@@ -0,0 +1,34 @@
+# Git Kata: Tagging of the commits
+## Setup:
+
+1. Run `. setup.sh`
+
+## Motivation
+
+[Tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) are convenient for
+keeping track of commits that bump a version number. One way of using them is
+together with branches, where we can tag commits in a branch if they bump the
+version number, e.g., a branch called `version/1.1` can have the tags `1.1.1`,
+`1.1.2` etc. Tags can also be used without branches.
+
+There are two types of tags. Annotated tags can contain additional information,
```

Comment:
As there are 2 types, you should probably describe both :-) and the differences...